book_config.py

```diff
@@ -195,8 +189,13 @@ def search_for_goodreads_url(self):
             self.goodreads_url_edit.setText('')
 
     def update_aliases(self):
+        if 'goodreads.com' not in self.goodreads_url_edit.text():
```

Comment:
More accurate (comment applies above as well, can ignore in both cases maybe as this is 100% pedantry!)
```if 'goodreads.com' not in urlparse.urlsplit(self.goodreads_url_edit.text()).netloc:```